### ISSUE TITLE
Another false positive

### ISSUE THREAD

**Issue by bunyk** [charithe/durationcheck#20]
```go
	daysInWeekend := 2
	weekendDuration := time.Duration(daysInWeekend) * 24 * time.Hour // durationcheck complains
	weekendDuration2 := time.Duration(daysInWeekend*24) * time.Hour // fine for durationcheck
```

`weekendDuration`, and `weekendDuration2` both contain the correct duration of 48h.
